addons/mail/static/src/components/messaging_menu/messaging_menu.xml

```diff
@@ -43,8 +43,8 @@
                                     customClass="id + '_mobileNewMessageInputAutocomplete'"
                                     isFocusOnMount="true"
                                     placeholder="mobileNewMessageInputPlaceholder"
-                                    select="_onMobileNewMessageInputSelect"
-                                    source="_onMobileNewMessageInputSource"
+                                    select="discuss.handleAddChatAutocompleteSelect.bind(discuss)"
+                                    source="discuss.handleAddChatAutocompleteSource.bind(discuss)"
```

Comment:
don't re-use the same handler, this is another element, so make new handler (internally they might call the same function if they do the same thing) (commit: reuse autocomplete function)